### PR TITLE
Remove ONNX consumed_inputs

### PR DESCRIPTION
**Pull request by bddppq**
https://github.com/onnx/onnx/pull/617

### CODE REVIEW

**Comment by bddppq** [comment]
DO NOT MERGE before reverting the onnx submodule back.

**Review by yinghai — APPROVED** [review]
Code looks good to me. Let's wait for the onnx submodule to be updated before merging.

**Comment by houseroad** [comment]
@caffe2bot retest this please

**Pull request merged**
